### ISSUE TITLE
Add Masks for Personalized Prompts

### ISSUE THREAD

**Issue by GetFractional** [boyueluzhipeng/GPT_CodeInterpreter#2]
One of the biggest downsides for me with ChatGPT is that I cannot create and save masks that have custom system prompts, which prevents me from using this with clients. I've used https://github.com/Yidadaa/ChatGPT-Next-Web to solve this.

I would love to see this integrated into your project. This would drive a ton of value to users, especially those looking to use this for business purposes.

**Specifically:**
1. Create a Mask to store a custom system prompt
2. Assign a LLM model
3. Configure LLM settings (either on a per mask basis like Next Web, or on a per thread basis like BetterGPT does)

**Comment by boyueluzhipeng** [comment]
Yes, I am also considering how to add a character similar to the mask function in ChatGPT-Next-Web that you mentioned. I plan to write more plugins in advance to implement different functionalities. Based on the characters, I will develop functions that are more closely related to their daily problem-solving needs.
